indexd/index/drivers/alchemy.py

```diff
@@ -633,6 +635,71 @@ def add(self,
 
             return record.did, record.rev, record.baseid
 
+    def add_blank_record(self, did=None, uploader=None, baseid=None):
+        """
+        Create a new blank record with only
+        """
+        with self.session as session:
+            record = IndexRecord()
+            base_version = BaseVersion()
+
+            if not baseid:
+                baseid = str(uuid.uuid4())
+            base_version.baseid = baseid
+
+            if did:
+                record.did = did
+            else:
+                new_did = str(uuid.uuid4())
+                if self.config.get('PREPEND_PREFIX'):
+                    new_did = self.config['DEFAULT_PREFIX'] + new_did
+                record.did = new_did
+
+            record.rev = str(uuid.uuid4())[:8]
+            record.baseid = baseid
+            record.uploader = uploader
+            
+            session.merge(base_version)
+            session.add(record)
+            session.commit()
+
+            return record.did, record.rev, record.baseid
+    
+    def update_blank_record(self, did, rev, size=None, hashes=None):
+        """
+        Create a new blank record with only
```

Comment:
docstring needs to be updated

Reply:
Fixed in 83f5b6b6dfc04b9eaa7d6502d320ae6c639ba1aa